pkgs/servers/monitoring/net-snmp/default.nix

```diff
@@ -14,37 +14,33 @@ stdenv.mkDerivation rec {
       inherit name sha256;
     };
   in [
-    (fetchAlpinePatch "CVE-2015-5621.patch" "05098jyvd9ddr5q26z7scbbvk1bk6x4agpjm6pyprvpc1zpi0y09")
-    (fetchAlpinePatch "fix-Makefile-PL.patch" "14ilnkj3cr6mpi242hrmmmv8nv4dj0fdgn42qfk9aa7scwsc0lc7")
     (fetchAlpinePatch "fix-includes.patch" "0zpkbb6k366qpq4dax5wknwprhwnhighcp402mlm7950d39zfa3m")
     (fetchAlpinePatch "netsnmp-swinst-crash.patch" "0gh164wy6zfiwiszh58fsvr25k0ns14r3099664qykgpmickkqid")
-    (fetchAlpinePatch "remove-U64-typedef.patch" "1msxyhcqkvhqa03dwb50288g7f6nbrcd9cs036m9xc8jdgjb8k8j")
-    ./CVE-2018-18065.patch
+    ./0001-no-network-tests.patch
```

Comment:
@cyounkins if we are not running the tests, do we even need this patch?

Reply:
IMO we should be moving towards enabling the tests. While not necessary right now, I think it makes sense to keep it in for when we touch it next.

Reply:
The patch is then essentially dead code, and we can't even be sure the patch works. I'd say let's only introduced patches once we actually need them, and keep builds as simple as possible.

Reply:
Especially since the `doCheck = false;` comment is "Tries to use networking", which suggests that the patch is indeed not doing its job…